p2p/memp2p/memp2p.go

```diff
@@ -0,0 +1,288 @@
+package memp2p
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go/p2p"
+)
+
+type MemP2PMessenger struct {
```

Comment:
Could you add comments for exported structures, functions, interfaces in non test files?

Reply:
Added comments to all structures and functions of the package.